lib/cob_index/indexer_config.rb

```diff
@@ -67,23 +67,23 @@
     247abcdefgnp
     740anp
                }.join(":"))
-to_field "title_added_entry_t", extract_marc_with_flank(%W{
+to_field "title_added_entry_t", extract_marc(%W{
   700gklmnoprst
   710fgklmnopqrst
   711fgklnpst
 
-                                             }.join(":"))
+                                             }.join(":")), wrap_begin_end
 to_field "title_sort", extract_marc("245abcfgknps", alternate_script: false, first: true)
 
 # Creator/contributor fields
-to_field "creator_t", extract_marc_with_flank("245c:100abcdejlmnopqrtu:110abcdelmnopt:111acdejlnopt:700abcdejqu:710abcde:711acdej", trim_punctuation: true)
+to_field "creator_t", extract_marc("245c:100abcdejlmnopqrtu:110abcdelmnopt:111acdejlnopt:700abcdejqu:710abcde:711acdej", trim_punctuation: true), delete_if(CORPORATE_NAMES), wrap_begin_end
 to_field "creator_facet", extract_marc("100abcdq:110abcd:111ancdj:700abcdq:710abcd:711ancdj", trim_punctuation: true), delete_if(CORPORATE_NAMES)
-to_field "creator_display", extract_creator
-to_field "contributor_display", extract_contributor
-to_field "creator_vern_display", extract_creator_vern
-to_field "contributor_vern_display", extract_contributor_vern
+to_field "creator_display", extract_creator, delete_if(CORPORATE_NAMES)
+to_field "contributor_display", extract_contributor, delete_if(Proc.new { |v| CORPORATE_NAMES.include?(JSON.parse(v)["name"]) })
```

Comment:
@dkinzer one more question -- why does contributor_display require a different approach for delete_if than creator_display, creadtor_vern_display, and contributor_vern_display? My exception is that they would all be handled the same way?

Reply:
@ebtoner it's because  the values generated by the `extract_contributor`  macro are structured json strings  and we can't do the easy  one to one comparison that is required for using  the `delete_if(CORPORATE_NAMES)` approach.